torch_np/tests/numpy_tests/core/test_scalarmath.py

```diff
@@ -483,13 +482,13 @@ def test_numpy_scalar_relational_operators(self):
                         "type %s and %s failed" % (dt1, dt2))
 
         #Unsigned integers
-        for dt1 in 'BHILQP':
+        for dt1 in 'B':
             assert_(-1 < np.array(1, dtype=dt1)[()], "type %s failed" % (dt1,))
             assert_(not -1 > np.array(1, dtype=dt1)[()], "type %s failed" % (dt1,))
             assert_(-1 != np.array(1, dtype=dt1)[()], "type %s failed" % (dt1,))
 
             #unsigned vs signed
-            for dt2 in 'bhilqp':
+            for dt2 in 'bhil':
```

Comment:
nb. All these tiny changes of things we don't support would be better factored out into their own PR to make reviews easier.